src/testdir/test_filetype.vim

```diff
@@ -2774,6 +2773,24 @@ func Test_make_file()
   filetype off
 endfunc
 
+func Test_map_file()
+  filetype on
+
+  " TI linker map file
+  call writefile(['******************************************************************************', '               TMS320C6x Linker Unix v7.4.24                   ', '******************************************************************************'], 'Xfile.map', 'D')
+  split Xfile.map
+  call assert_equal('lnkmap', &filetype)
+  bwipe!
+
+  " TI linker map file
```

Comment:
Shouldn't it say `Map (UMN mapserver config file)` here?

Reply:
Sorry for a typo.

Reply:
Fixed here #16320